### PR TITLE
Fix press of Enter on post title.

### PR DESCRIPTION
**Pull request by SergioEstevao**
## Description

Make sure onEnterPress is called on the right prop of RichText

## How has this been tested?

You can this PR on GB-mobile to test it: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1057
## Types of changes

 Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate.

### CODE REVIEW

File: packages/editor/src/components/post-title/index.native.js [hunk 1]
```diff
@@ -100,7 +100,8 @@ class PostTitle extends Component {
 					} }
 					placeholder={ decodedPlaceholder }
 					value={ title }
-					onSplit={ this.props.onEnterPress }
+					onSplit={ () => { } }
```
Comment (hypest) [inline packages/editor/src/components/post-title/index.native.js]: Do we need to set the `onSplit` to something? Can we just not set the prop at all?

**Comment by SergioEstevao** [comment]
In order for RichText to send the onReplace callback, both onSplit and
onReplace need to be defined.

**Comment by SergioEstevao** [comment]
Here is the relevant line on RichText: https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/rich-text/index.native.js#L196

File: packages/editor/src/components/post-title/index.native.js (hunk 1 above)
Comment (hypest) [reply packages/editor/src/components/post-title/index.native.js]: > https://github.com/WordPress/gutenberg/pull/15944#issuecomment-498146491
>
>  https://github.com/WordPress/gutenberg/pull/15944#issuecomment-498166693

Ah, I see 😿

**Review by koke — APPROVED** [review]

**Pull request merged**